subsys/bluetooth/audio/shell/audio.h

```diff
@@ -182,6 +198,22 @@ int cap_ac_unicast(const struct shell *sh, const struct bap_unicast_ac_param *pa
 #endif /* CONFIG_BT_BAP_UNICAST_CLIENT */
 #endif /* CONFIG_BT_BAP_UNICAST */
 
+static inline uint8_t get_chan_cnt(enum bt_audio_location chan_allocation)
```

Comment:
We may consider making this a common function; it is used several other places

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/69617